File: app/src/main/java/com/explead/seasons/winter/interfaces/OnActionMoveCubeListener.java
package com.explead.seasons.winter.interfaces;

import com.explead.seasons.winter.logic.WinterCube;

public interface OnActionMoveCubeListener {
    void onNotMove();
    void onGoOnCell();
    void onArrow();
    void onCubeOnPlace(WinterCube cube);
}

File: app/src/main/java/com/explead/seasons/winter/interfaces/OnActionMoveCubeListener.kt
package com.explead.seasons.winter.interfaces

import com.explead.seasons.winter.logic.WinterCube

interface OnActionMoveCubeListener {
    fun onNotMove()
    fun onGoOnCell()
    fun onArrow()
    fun onCubeOnPlace(cube: WinterCube)
}
